### PR TITLE
Update draft-ietf-sidrops-rpki-prefixlist.xml

### PR DESCRIPTION
**Pull request by gih900**
changes as proposed by Ties de Kock (with a couple of my own added in)

### CODE REVIEW

**Review by job — APPROVED** [review]

File: draft-ietf-sidrops-rpki-prefixlist.xml [hunk 1]
```diff
@@ -270,12 +270,12 @@
...
       <t>
         Relying Parties are warned that the data in a Signed Prefix List is self-asserted by the AS holder.
-        There is no implied authority from any IP prefix holder that grants the AS permission to originate a route for any prefixes.
+        There is no implied authority in a Signed Prefix List that any IP prefix holder has granted the AS permission to originate a route for any of the listed prefixes.
         Such an authority is separately conveyed in the RPKI as a ROA.
       </t>
       <t>
-        While a one-time-use EE certificate must only be used to generate and sign a single Signed Prefix List object, CAs are not technically restricted from generating and signing multiple different Signed Prefix List objects with a single key pair.
-        Any Signed Prefix List objects sharing the same EE certificate cannot be revoked individually.
+        While a one-time-use EE certificate are only be used to generate and sign a single Signed Prefix List object, CAs are not technically restricted from generating and signing multiple different Signed Prefix List objects with a single key pair.
```
Comment (ties) [inline draft-ietf-sidrops-rpki-prefixlist.xml:287]: This first sentence does not flow very well;

```suggestion
        While the key pairs of one-time-use EE certificates are only used to generate and sign a single Signed Prefix List object, CAs are not technically restricted from generating and signing multiple different Signed Prefix List objects with a single key pair.
```

Comment (job) [reply draft-ietf-sidrops-rpki-prefixlist.xml:287]: Thanks, I worded it slightly different: https://github.com/job/draft-sidrops-rpki-prefixlist/commit/9732c6daf43fc2e0c8bc1ecabd8533613b5eaf7a

**Comment by ties** [comment]
If we are bikeshedding (and I feel like I am): in my mind the private key is ephemeral. The certificate and public key are very much long lived.

**Comment by job** [comment]
You can:
- recycle the key pair
- recycle the certificate (and its serial)
- combine the above

The goal is to recommend to not recycle either the key pair or the certificate (serial)

**Pull request merged**